contracts/chain-adapters/WorldChain_Adapter.sol

```diff
@@ -32,6 +32,7 @@ contract WorldChain_Adapter is CrossDomainEnabled, AdapterInterface, CircleCCTPA
     WETH9Interface public immutable L1_WETH;
 
     IL1StandardBridge public immutable L1_STANDARD_BRIDGE;
+    IOpUSDCBridgeAdapter public immutable L1_OP_USDC_BRIDGE;
```

Comment:
Should this be name have Cctp in it somehow since it's a cctp system contract not an op stack one?

Reply:
It's not actually a CCTP contract, it's a third-party implementation of a Circle interface that will support a migration to CCTP, but from what I can see the actual deposit/withdrawal methods are implementation-defined. So I believe this interface is actually hard-coupled to OP chains that choose to use the interface.